lib/yardstick/ordered_set.rb

```diff
@@ -17,7 +17,7 @@ class OrderedSet
     def initialize(entries = nil)
       @entries = []
       @index   = {}
-      merge(entries || [])
+      merge(entries || self)
```

Comment:
I realize how this works, but it's a little weird. What about changing it to something more explicit, like:

```ruby
merge(entries) if entries
```

Although one thing this doesn't protect against is cases where the caller passes in `nil` explicitly. Often I've defined an `Undefined` constant in the root project namespace, and used that to represent the default argument, eg:

```ruby
module Yardstick
  Undefined = Object.new.freeze

  # ...

  class OrderedSet
    def initialize(entries = Undefined)
      @entries = []
      @index   = {}
      merge(entries) unless entries.equal?(Undefined)
    end
  end
end
```